src/components/backgroundselector/style/backgroundselector.less

```diff
@@ -1,256 +1,263 @@
-      background-color: red;
+    .ga-bg-layer-bt {
+      width: 96px;
+      height: 64px;
+      bottom: 52px;
+      right: 10px;
+      background: #ffffff data-uri('swissimage-rect.png') no-repeat center;
+     
+      @media (max-width: @screen-tablet) {
+        width: 48px;
+        height: 32px;
+        bottom: 24px;
+        background-size: 44px 28px;
+      }
     }
-    @media (max-width: @screen-tablet) {
-      width: 48px;
-      height: 32px;
-      bottom: 24px;
-      background-size: 44px 28px;
+
+    .ga-bg-layer-bt-close {
+      width: 20px;
+      background-image: none;
+
+      .ga-bg-layer-bt-text {
+        display: none;
+      }
+
+      .icon-chevron-sign-right {
+        display: block;
+      }
     }
-  }
 
-  .ga-bg-highlight {
-    background-color: rgba(255, 0, 0, 1);
-  }
+    .ga-bg-layer {
+      width: 96px;
+      height: 64px;
+      bottom: 52px;
+      right: 10px;
+      background-color: rgba(255, 255, 255, 1);
+      &:hover {
+        background-color: red;
+      }
+      @media (max-width: @screen-tablet) {
+        width: 48px;
+        height: 32px;
+        bottom: 24px;
+        background-size: 44px 28px;
+      }
+    }
 
-  .ga-pixelkarte-farbe {
-    background-image: data-uri('pixelkarte-farbe-rect.png');
-  }
+    .ga-bg-highlight {
+      background-color: rgba(255, 0, 0, 1);
+    }
 
-  .ga-pixelkarte-grau {
-    background-image: data-uri('pixelkarte-grau-rect.png');
-  }
+    .ga-pixelkarte-farbe {
+      background-image: data-uri('pixelkarte-farbe-rect.png');
+    }
 
-  .ga-swissimage {
-    background-image: data-uri('swissimage-rect.png');
-  }
+    .ga-pixelkarte-grau {
+      background-image: data-uri('pixelkarte-grau-rect.png');
+    }
 
-  .ga-voidLayer {
-    background-image: data-uri('voidLayer-rect.png');
-  }
+    .ga-swissimage {
+      background-image: data-uri('swissimage-rect.png');
+    }
 
-  .ga-bg-layer-0 {
-    right: 35px;
-  }
+    .ga-voidLayer {
+      background-image: data-uri('voidLayer-rect.png');
+    }
 
-  .ga-bg-layer-1 {
-    right: 136px;
-    @media (max-width: @screen-tablet) {
-      right: 88px;
+    .ga-bg-layer-0 {
+      right: 35px;
     }
-  }
 
-  .ga-bg-layer-2 {
-    right: 237px;
-    @media (max-width: @screen-tablet) {
-      right: 141px;
+    .ga-bg-layer-1 {
+      right: 136px;
+      @media (max-width: @screen-tablet) {
+        right: 88px;
+      }
     }
-  }
 
-  .ga-bg-layer-3 {
-    right: 338px;
-    @media (max-width: @screen-tablet) {
-      right: 194px;
+    .ga-bg-layer-2 {
+      right: 237px;
+      @media (max-width: @screen-tablet) {
+        right: 141px;
+      }
+    }
+
+    .ga-bg-layer-3 {
+      right: 338px;
+      @media (max-width: @screen-tablet) {
+        right: 194px;
+      }
     }
   }
 }
 
 .ga-3d-active {
-  .ga-disable3d {
-    cursor: default;
-    background-color: gray;
-    color: gray;
-    &:hover {
+
+  [ga-background-selector] {
+
+    .ga-disable3d {
+      cursor: default;
```

Comment:
yes @loicgasser cursor: not-alowed is more indicated.
yes @cedricmoullet @davidoesch I put it on the right.